v4/feature/trade/src/main/java/exchange/dydx/trading/feature/trade/trigger/components/DydxTriggerOrderCtaButtonViewModel.kt

```diff
@@ -65,10 +65,10 @@ class DydxTriggerOrderCtaButtonViewModel @Inject constructor(
                 DydxTriggerOrderCtaButtonView.State.Disabled(buttonTitle)
             },
             ctaAction = {
-                router.navigateBack()
                 abacusStateManager.commitTriggerOrders { _ ->
                     // order status will be shown from PresentationProtocol.showToast()
                 }
+                router.navigateBack()
```

Comment:
does this need to go in the callback?

Reply:
oh nvm we only care about submission

Reply:
The callback gets called multiple times, one for each order submission.